src/mpecopt_phase_ii.m

```diff
@@ -379,8 +379,8 @@
         cpu_time_nlp_k_l = toc(t_nlp_start);
         x_k_multi = full(results_nlp.x);
         lambda_x_k = full(results_nlp.lam_x);
+        [stats.multiplier_based_stationarity, ~] = determine_multipliers_based_stationary_point(x_k_multi,lambda_x_k,dims,settings);
     end
-    [stats.multiplier_based_stationarity, ~] = determine_multipliers_based_stationary_point(x_k_multi,lambda_x_k,dims,settings);
```

Comment:
it was fine to have the check here, i.e., as it was.
the inner if just creates a TNLP if BNLPs are used for NLP subproblems
the outer checks will be the stat point computed. if it is already a TNLP subprob, then the inner if is not needed.

Reply:
the absence of x_k_multi might be a problem for directly TNLP.